engine/access/rest/websockets/controller.go

```diff
@@ -4,100 +4,225 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"sync"
+	"time"
 
 	"github.com/google/uuid"
 	"github.com/gorilla/websocket"
 	"github.com/rs/zerolog"
 
 	dp "github.com/onflow/flow-go/engine/access/rest/websockets/data_provider"
 	"github.com/onflow/flow-go/engine/access/rest/websockets/models"
-	"github.com/onflow/flow-go/engine/access/state_stream"
-	"github.com/onflow/flow-go/engine/access/state_stream/backend"
 	"github.com/onflow/flow-go/utils/concurrentmap"
 )
 
+const (
+	// PingPeriod defines the interval at which ping messages are sent to the client.
+	// This value must be less than pongWait.
+	PingPeriod = (PongWait * 9) / 10
+
+	// PongWait specifies the maximum time to wait for a pong message from the peer.
+	PongWait = 10 * time.Second
```

Comment:
Wouldn't it be better to place it in `websockets.Config` type?

Reply:
I believe it would make sense to add this to the `websockets.Config` only if the operator has the ability to customize it. Since such customization does not seem necessary in this case, it may be more reasonable to avoid moving it to `websockets.Config`.

Reply:
But I agree with your point that it’s better to have all configuration values in one place. To address this, I’ve moved these constants to the `config.go` file ([d58446d](https://github.com/onflow/flow-go/pull/6757/commits/d58446d6f2d1082b0f121f49c3d3d303f59d5ce6)).